lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,47 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery:
+    description:
+      - A dictionary representing the recovery actions you want to set.
+    suboptions:
+      reset_fail_count_after:
+        description:
+          - Reset failure counter after this number of seconds.
+        type: int
+      on_first_failure:
+        description:
+          - Action to take on first failure of the service.
+          - Only restart and reboot are supported value.
```

Comment:
I suggest using the 'choices' and 'default' documentation elements here as they will format better in the module documentation page. 
for example, below 'description' add

```
        description:
          - Action to take on first failure of the service.
        choices: [ restart, reboot, no_action ]
        default: no_action
```

Reply:
Thanks for the suggestion, I changed it!